frontend/pages/DetailedViewPage.js

```diff
@@ -1,5 +1,5 @@
 import React, {Component} from 'react';
-import {searchByUPC} from "../utils/api";
+import {addPrice, searchByUPC} from "../utils/api";
 import { View, ActivityIndicator, StyleSheet} from 'react-native';
```

Comment:
I think it is a better to have a consistent styling for number of white spaces between brackets.
Either
```
import {View, ActivityIndicator, StyleSheet} from 'react-native';
```
or
```
import { View, ActivityIndicator, StyleSheet } from 'react-native';
```